### ISSUE TITLE
Version parsing error on Ubuntu

### ISSUE THREAD

**Issue by cpheps** [newrelic/nri-postgresql#15]
##### Description of the problem

The follow log is printed when trying to parse the version of PostgreSQL when on Ubuntu

```
stderr="[ERR] Metrics collection failed: error collecting version number: Invalid character(s) found in minor number \"4 (Ubuntu 10\"\n"
```

Looks as though it is expecting semantic versioning but Ubuntu version return strings in the last position, shown [here](https://packages.ubuntu.com/search?keywords=postgresql).

##### OS

- [ ] All of them
- [ ] Amazon Linux, all versions
- [ ] CentOS, version 6 or higher
- [ ] Debian, version 7 ("Wheezy") or higher
- [ ] Red Hat Enterprise Linux (RHEL), version 6 or higher
- [X] Ubuntu, versions 12.04, 14.04, and 16.04 (LTS  versions)
- [ ] Windows Server, 2008 and 2012 and their service packs

**Comment by cpheps** [comment]
@sonaikar It is but we were not aware this same bug exists for Debian. Looks like we will need further investigation across linux distributions.

**Comment by sonaikar** [comment]
Thanks for closing on this.  Is this version part of apt-get packages? 
I tried to build it manually but failed with error, 

```
=== postgresql === [ deps ]: Installing package dependencies required by the project...
Error: Package "/root/newrelic/nri-postgresql-1.0.2" not a go package or not in GOPATH.
govendor sync
	Ensures the contents of the vendor folder matches the vendor file.
	Options:
		-n           dry run, print out action only
		-insecure    allow downloading over insecure connection
		-v           verbose output
Makefile:37: recipe for target 'deps-only' failed
make: *** [deps-only] Error 2
```

Any suggestion to setup GOPATH properly?

**Comment by cpheps** [comment]
I haven't released it yet will do tomorrow morning, I'll let you know when we do so. As for the GOPATH thing it's because your trying to compile a Go program in the incorrect path. I think once we release it tomorrow that shouldn't be an issue.

**Comment by cpheps** [comment]
@sonaikar We've officially released Postgres 1.0.2

**Comment by sonaikar** [comment]
It worked as expected. Thanks a ton @cpheps .
